### ISSUE TITLE
bin

### ISSUE THREAD

**Issue by darlanmendonca** [nanohop/sketch-to-react-native#7]
nice project!! really

my suggestion is create a bin to module, instead of:

```sh
node convert.js input-file.svg
```

to use like below

```sh
sketch-to-react-native input-file.svg
```

what u think? if you like, I can do a PR

**Comment by chrisachard** [comment]
@darlanmendonca That would be great!

**Comment by darlanmendonca** [comment]
Nice!! @chrisachard help me on PR please.

**Comment by chrisachard** [comment]
Closing as fixed in PR #10 - but I still want to get it on npm as suggested, so I'll open a new issue
